packages/apib-serializer/test/adapter-test.js

```diff
@@ -47,6 +47,30 @@ describe('API Blueprint serializer adapter', () => {
         return done();
       });
     });
+
+    it(`serializes synchronously ${path.basename(file)}`, (done) => {
```

Comment:
Since this isn't an async API, perhaps it would be clearer to omit `done`. I think the test would be simpler without it since you won't need to wrap the blocks in try/catch as throwing from the test would trigger test failure.

Reply:
Absolutely

Reply:
Addressed in cd072b1